shell/common/shorebird/BUILD.gn

```diff
@@ -31,6 +31,15 @@ source_set("shorebird") {
   ]
 
   include_dirs = [ "//flutter/updater" ]
+
+  if (host_os == "win" && target_os == "win") {
+    if (target_cpu == "x64") {
+      libs = [
+        "userenv.lib",
```

Comment:
Why is this needed in two places?  Should we have a shared target both of these depend on?

Reply:
Removed from the other BUILD file